### ISSUE TITLE
Create evaluation script for model predictions.

### ISSUE THREAD

**Issue by rloganiv** [ucinlp/covid19-backend#15]
# Requirements
Create an evaluation script: `scripts/ml/evaluate.py`

The script will need to take as an input:
1. Misconception data
2. A trained model (in the long term we want to support any `Detector` but in the short term it will suffice to only evaluate `SentenceBertClassifier` style models)
3. [Annotations](https://docs.google.com/spreadsheets/d/1cmIpK8g_lLVBqKB-rT06B6suNq9KtGZnOu38v591tlE/edit?usp=sharing)

The script should output the following evaluation metrics:
1. Accuracy of classification models (e.g., SentenceBERT / anything trained to perform NLI). This is a straightforward computation - the `misconception` and `tweet` columns provide the model output and the `pos/neg/na` column provides the gold label.
2. [Hits@k](https://stackoverflow.com/questions/58796367/how-is-hitsk-calculated-and-what-does-it-mean-in-the-context-of-link-prediction) and [mean reciprocal rank](https://en.wikipedia.org/wiki/Mean_reciprocal_rank) of all models. Given an input (e.g., tweet, reddit post, etc.) compute the positive match score between that input and *every* misconception. The metrics can then be computed by ranking the misconceptions by score and identifying the rank of the true misconception being expressed.
3. precision-recall curve. Ultimately we will need to use a cutoff to predict whether or not a tweet is truly expressing a misconception, having a precision-recall curve will be useful for identifying the best score cutoff.

**Comment by rloganiv** [comment]
@atleastbeth3 I've uploaded model weights for a `SentenceBertClassifier` trained on MultiNLI to the google drive [here](https://drive.google.com/open?id=1I9PoNaM7QQRENrSyBgzJpcrOyaqaU_X0).

**Comment by rloganiv** [comment]
You will need to work on the https://github.com/ucinlp/covid19-backend/tree/model-dump branch. It would be especially helpful if you would make pull requests instead of directly making commits so that I can provide feedback before changes are made. If you use the terminal you can use the following commands to create your working branch:
```{bash}
# Update you local repo
git fetch --all 

# Checkout the model-dump branch
git checkout model-dump

# Create another branch off of this branch for you to work on. Let's call it md-tthossai.
git checkout -b md-tthossai
```
Whenever you want to make your work available to me or another group member,  remember to:
```
# Make sure you are on the right branch. If you run the following command it will tell you
# which branch you are working on. Make sure it says md-tthossai
git status

# Add your work
git add [whatever files you've created or changed]

# Commit your work
git commit -m "[A message describing what you've added]"

# Push to github
git push origin md-tthossai
```
Then I'll walk you through the process of submitting a pull request. This process can be pretty confusing if you are new to Git, feel free to ping me if you have any questions---if you are unsure about anything make sure not to do the final push step.

**Comment by rloganiv** [comment]
Closed by #48
